examples/text-generation/run_generation.py

```diff
@@ -312,6 +302,35 @@ def setup_parser(parser):
         help="Run multi card with the specified parallel strategy. Choices are 'tp' for Tensor Parallel Strategy or 'none'.",
     )
 
+    parser.add_argument(
+        "--run_partial_dataset",
+        action="store_true",
+        help="Run the inference with dataset for specified --n_iterations(default:5)",
+    )
```

Comment:
Where is this used?

Reply:
[examples/text-generation/run_generation.py](https://github.com/HabanaAI/optimum-habana-fork/blob/74d5dc8a958666a04cc73a8ba25db0cd65f1a7f4/examples/text-generation/run_generation.py#L866)

Reply:
It's not used in this PR so I don't think it should be part of it. Please open a new PR to add this new argument to the text-generation example.